content/guides/building-dockerfile-images-with-buildkit.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Building Dockerfile Images with Buildkit
+description: |
+  This guide shows how to install buildkit to allow you to build Dockerfile images with Unikraft.
+---
+Unikraft uses BuildKit, along with Dockerfiles, to construct the root filesystem used by applications.
+
+
+You can install BuildKit by either running it natively via systemd or in a
+container. Please refer to [BuildKit's installation
+instructions](https://github.com/moby/buildkit) for more information.
+
+To use BuildKit with Docker on your system, you first need to install Docker
+following the [official
+instructions](https://docs.docker.com/engine/install/). This can be either
+[Docker Engine](https://docs.docker.com/engine/) or [Docker
+Desktop](https://docs.docker.com/desktop/).
```

Comment:
Use one and only one sentence per line. When a period appears (`.`), you start a newline. We have the documentation guidelines buried somewhere, we will dig them out.

Reply:
Thanks, will fix.